docs/Button.md

```diff
@@ -70,12 +70,13 @@ This widget provides default styling for the following delite themes:
 
 The base class for a button is `d-button`.
 
-The appearance of a button can be customized by adding extra classes to it:
+The `boostrap theme` (default) provide extra classes to customize the button:
+* `d-button-success`
+* `d-button-info`
+* `d-button-warning`
+* `d-button-danger`
 
```

Comment:
maybe just a couple of words on what it is doing by default? changing color scheme?